build/build.js

```diff
@@ -4,25 +4,95 @@ var semver = require('semver');
 var fs = require('fs');
```

Comment:
Still missing:
* signing; but because we didn't have it before I considered it was not needed in this PR